Cargo.lock

```diff
@@ -8906,8 +8906,6 @@ dependencies = [
  "thiserror",
  "tokio",
  "zk_evm 1.3.3 (git+https://github.com/matter-labs/era-zk_evm.git?tag=v1.3.3-rc2)",
```

Comment:
Is it necessary to keep 1.3.3? 
@StanislavBreadless you are also working on something like it

Reply:
I've raised this topic before, it's out of scope for this PR.